numpy/random/_generator.pyx

```diff
@@ -454,6 +454,21 @@ cdef class Generator:
         >>> x-y
         0.08 # may vary
 
+        The distribution can be visualized as follows:
+
+        >>> import numpy as np
+        >>> import matplotlib.pyplot as plt
+        >>> import scipy.stats as stats
+        >>> l = 2
```

Comment:
Rename `l` to `scale`?

Reply:
@eendebakpt This change should actually be here #25356, in the literature the parameter is usually called lambda, which doesn't work obviously. In wikipedia they call it a "rate parameter". I could change it to rate. I would know what scale is supposed to mean.